go/webpack/cbor_format_test.go

```diff
@@ -31,10 +31,14 @@ func TestWriteCBOR(t *testing.T) {
 	pack := Package{
 		parts: []*PackPart{
 			&PackPart{
-				url:            staticUrl("https://example.com/index.html?query"),
-				requestHeaders: HTTPHeaders{},
-				status:         200,
+				requestHeaders: HTTPHeaders{
+					httpHeader(":method", "GET"),
```

Comment:
I feel I should have asked before, but what is potential purpose to keep 'method' here? It seems to add to confusion because it's unclear how would that be processed by UA and what the method would exactly mean.